code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -29,6 +31,7 @@
 /obj/item/reagent_containers/food/snacks/human/burger
```

Comment:
Unrelated, but I hate this fuckin path

Reply:
Hopefully in a later PR I or someone else can get this to work properly so the path can be killed.